styles/app/components/sidebar/sidebar.module.css

```
@@ -36,15 +29,9 @@
 		display: flex;
 		justify-content: space-between;
 		align-items: center;
-		padding-top: 15px;
+		padding: 15px 20px 20px 20px;
```

Tu peux le remplacer par ça 

```suggestion
		padding: 15px 20px 20px;
```

Cela correspond à `padding: left y right`